docs/podman-run.1.md

```diff
@@ -501,7 +501,7 @@ This works for both background and foreground containers.
 
 **--network**, **--net**=*mode*
 
-Set the Network mode for the container:
+Set the Network mode for the container. Invalid if using **--dns**, **--dns-option**, or **--dns-search** with **--network** that is not set to host.
```

Comment:
This should be a Node,  I think the --net and --network can only be used with without none, host or container:NAME.